site/global/sections/footer.json

```diff
@@ -48,7 +48,7 @@
                     {
                         "label": "Locations And Hours",
                         "type": "external",
```

Comment:
Can we start using the `Link` resource shape + the `link` function? 

There was [recently some documentation added for this](https://square.github.io/custom-sites-docs/template-syntax/functions/link) and we should document the shape of the `Link` resource in its own section (it's the same as the function doc)

In this case, it'd be:

```
{
  "type": "storeLocator"
}
```

and ideally consumed like

```twig
<a href="{{ link(payload) }}">Store locator</a>
```